dev/local/Makefile

```diff
@@ -0,0 +1,136 @@
+# Docker control panel for delphi-epidata development
+#
+# Usage: make <command> [pdb=1] [test-subdir=<test-subdir>]
+#
+# Assumes you have installed your environment using
+# delphi-epidata/dev/local/install.sh.
+#
+# Checks for the delphi-net bridge and creates if it doesn't exist.
+#
+# Creates all prereq images (delphi_database, delphi_python) only if they don't
+# exist. If you need to rebuild a prereq, you're probably doing something
+# complicated, and can figure out the rebuild command on your own.
+# 
+# 
+# Commands:
+#            
+#   web:     Stops currently-running delphi_web_epidata instances, if any.
+#            Rebuilds delphi_web_epidata image.
+#            Runs image in the background and pipes stdout to a log file.
+# 
+#   db:      Stops currently-running delphi_database_epidata instances, if any.
+#            Rebuilds delphi_database_epidata image.
+#            Runs image in the background and pipes stdout to a log file.
+#            Blocks until database is ready to receive connections.
+# 
+#   python:  Rebuilds delphi_web_python image. You shouldn't need to do this
+#            often; only if you are installing a new environment, or have
+#            made changes to delphi-epidata/dev/docker/python/Dockerfile.
+#
+#   all:     Runs the commands 'web' 'db' and 'python'.
+#
+#   test:    Runs test and integrations in delphi-epidata. If test-subdir 
+#            optional arg is provided, then only the tests in that subdir
+#            are run.
+#
+#   clean:   Cleans up dangling Docker images.
+#
+# 
+# Optional arguments:
+#   pdb=1         Drops you into debug mode upon test failure, if running tests.
+#   test-subdir=  Only runs tests in the directory provided here, e.g.
+#                 repos/delphi/delphi-epidata/tests/acquisition/covidcast
+
+
+# Set optional argument defaults
+ifdef pdb
+	override pdb=--pdb
+else
+	pdb=
+endif
+
+ifdef test-subdir
+else
+	test-subdir=repos/delphi/delphi-epidata/tests repos/delphi/delphi-epidata/integrations
+endif
+
+# Set your shell here
+SHELL:=/bin/sh
+
+# Get the Makefile's absolute path: https://stackoverflow.com/a/324782/4784655
+# (if called from symlink, the path is the location of the symlink)
+NOW:=$(shell date "+%Y-%m-%d")
+CWD:=$(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+LOG_WEB:=$(CWD)docker-logs/delphi_web_epidata_$(NOW).log
+LOG_DB:=$(CWD)docker-logs/delphi_database_epidata_$(NOW).log
```

Comment:
Why do these need to specify CWD?

Reply:
I'm just in the habit of specifying full paths at this point. Some tools need full, some don't 🤷.